src/plugins/jsepAssignment.js

```diff
@@ -0,0 +1,65 @@
+const PLUS_CODE = 43; // +
+const MINUS_CODE = 45; // -
+
+export default {
+	name: 'jsepAssignment',
+
+	init(jsep) {
+		// Assignment and Update support
+		const assignmentOperators = new Set([
+			'=',
+			'*=',
+			'**=',
+			'/=',
+			'%=',
+			'+=',
+			'-=',
+			'<<=',
+			'>>=',
+			'>>>=',
+			'&=',
+			'^=',
+			'|=',
+		]);
+		const updateOperators = [PLUS_CODE, MINUS_CODE];
+		const updateNodeTypes = [jsep.IDENTIFIER, jsep.MEMBER_EXP];
+		assignmentOperators.forEach(op => jsep.addBinaryOp(op, 0.9));
```

Comment:
Why 0.9?

Reply:
I looked up the [operator precedence table](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#table), but I just realized I read the precedence backward because assignment operators are lowest priority (3) so I'll update...